src/models/course.ts

```diff
@@ -32,4 +32,20 @@ export interface CourseInterface extends Document {
 
 const Course = dynamoose.model<CourseInterface>('Course', courseSchema);
 
+function CreateCourse({
```

Comment:
Great question - I can't remember haha. Maybe I added it because Semester's contain arrays of Courses and I thought I might need courses?! I'll remove it from my PR since it isn't used!